docs/configuration/overview.mdx

```diff
@@ -228,3 +228,26 @@ const MyClientComponent: React.FC = () => {
 Payload collects **completely anonymous** telemetry data about general usage. This data is super important to us and helps us accurately understand how we're growing and what we can do to build the software into everything that it can possibly be. The telemetry that we collect also help us demonstrate our growth in an accurate manner, which helps us as we seek investment to build and scale our team. If we can accurately demonstrate our growth, we can more effectively continue to support Payload as free and open-source software. To opt out of telemetry, you can pass `telemetry: false` within your Payload config.
 
 For more information about what we track, take a look at our [privacy policy](/privacy).
+
+## CORS
+
+CORS can be configured with either a whitelist array of URLS to allow CORS requests from, a wildcard string (`*`) to accept incoming requests from any domain, or a object with the following properties:
```

Comment:
We should show the example having an array of strings on the `cors` property direclty as that covers most uses and is the most simple to understand.